code/src/components/ThoughtList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { formatRelative } from 'date-fns';
+
+const ThoughtList = ({thought, handleLikes}) => {
+
+    return (
+    <section className='thoughtlist-wrapper'>
+         <form className='message-card' >
+            <p className='messages' key={thought._id}>{thought.message}</p>
+            
+            <div className="likedate-wrapper">
+                
+                <div className="like">
+                    <button 
+                        className={thought.hearts > 0 ? 'button-heart clicked' : 'button-heart'}
```

Comment:
I added the color directly in the code, but I think this is a really nice solution - to style it in CSS and just pick up the "clicked"-style